3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -1265,6 +1269,18 @@ def get_hdfs_expunge_step
       def get_processing_manifest(targets)
         targets[:ENRICHED_EVENTS].select { |t| not t.data[:processingManifest].nil? }.map { |t| t.data.dig(:processingManifest, :amazonDynamoDb, :tableName) }.first
       end
+
+      Contract ArrayOf[Iglu::SelfDescribingJson], String => Hash
+      def self.get_rdb_shredder_target(targets, shred)
```

Comment:
This function is pure, you could add a unit test.
You can find some examples [here](https://github.com/snowplow/snowplow/tree/master/3-enrich/emr-etl-runner/spec/snowplow-emr-etl-runner)